### ISSUE TITLE
Cardinal pieces no longer have drop shadow, don't match web version

### ISSUE THREAD

**Issue by blamarche** [veloce/lichobile#1283]
**Describe the bug**
Sometime in the last week or two of updates, the Cardinal piece graphics set removed the drop shadows, and no longer is the same as the web version. This was the only piece set with drop shadows, meaning there no longer any option for mobile app users to have a shadows on pieces.

**To Reproduce**
Go to options, set Piece set to Cardinal. View or play a game, pieces have no drop shadow

**Expected behavior**
Cardinal set should have drop shadows in parity with web version

**Screenshots**
Mobile:
![lichess](https://user-images.githubusercontent.com/623830/85064155-22217a80-b179-11ea-97e2-92f21661516b.jpeg)

Web:
![lichess2](https://user-images.githubusercontent.com/623830/85064334-6d3b8d80-b179-11ea-8de2-dfb3476938a2.jpg)

**Smartphone (please complete the following information):**
 - Device: Zenfone 5z
 - OS: Android
 - Browser N/a
 - Version 10

**Additional context**
I suppose other than just correcting Cardinal for parity, a feature request to enable optional drop shadows on all piece sets would be a more ideal, general purpose solution.

**Comment by greatwolf** [comment]
yea this was intentionally removed due to reported performance issues it causes. See #1241

**Comment by blamarche** [comment]
Ah... this would be a great candidate for an option then. My phone never showed any signs of lag. I'm going to load up the previous version and test just to make sure I didn't miss something. Could you point me to the code where this was changed/removed? Perhaps I can make a pull request where its added as an option.

**Comment by LichessUser** [comment]
Yes, make it to add as option. It would be great for both the high end and the low end devices.
Thank you.

**Comment by blamarche** [comment]
I noticed the shadows were actually in the svg files for the cardinal pieces. Its likely possible to inject the shadow filter svg tags dynamically as they are loaded or rendered, but some direction on where in the code you think would be best to do this would be great. @veloce

**Comment by greatwolf** [comment]
See commit 0adb206f35781149b4a71f6c6339c377f9495724 that removed drop shadows. It's possible to do drop shadows via CSS instead of svg. That would allow it be applied to other piece sets.
